frontend/amundsen_application/api/metadata/v0.py

```diff
@@ -525,8 +459,13 @@ def _log_update_table_tags(*, table_key: str, method: str, tag: str) -> None:
         _log_update_table_tags(table_key=table_key, method=method, tag=tag)
 
         metadata_status_code = _update_metadata_tag(table_key=table_key, method=method, tag=tag)
-        search_status_code = _update_search_tag(
-            key=table_key, resource_type=ResourceType.Table, method=method, tag=tag)
+        search_method = method if method == 'DELETE' else 'POST'
+        search_status_code = update_search_field(key=table_key,
+                                                 resource_type=ResourceType.Table.name.lower(),
+                                                 field='tag',
+                                                 value=tag,
+                                                 operation='add',
```

Comment:
kinda odd to do operation='add' and method=delete no?

Reply:
this all needs to change based on the feedback from search I implemented over last week